linkerd/proxy/transport/src/listen.rs

```diff
@@ -28,7 +28,7 @@ pub trait Bind<T> {
     fn bind(self, params: &T) -> Result<Bound<Self::Incoming>>;
 }
 
-pub type Bound<I> = (Local<ServerAddr>, I);
+pub type Bound<I> = (Local<ServerAddr>, Option<Local<ServerAddr>>, I);
```

Comment:
Do we need too include this when it's not used?

My expectation is that we can use this type alias when dealing with basic bind type, but that dual module should define its own
```rust
pub type Bound<I> = (Local<ServerAddr>, Option<Local<ServerAddr>>, I);
```

This where there's no ambiguity in the type system, no unwraps, etc.